### ISSUE TITLE
Release MonoArt on Hugging Face

### ISSUE THREAD

**Issue by NielsRogge** [Quest4Science/MonoArt#1]
Hi @hzxie 🤗

I'm Niels and work as part of the open-source team at Hugging Face. I discovered your work through Hugging Face's daily papers as yours got featured: https://huggingface.co/papers/2603.19231.
The paper page lets people discuss about your paper and lets them find artifacts about it (your models for instance), you can also claim the paper as yours which will show up on your public profile at HF, add Github and project page URLs.

Would you like to host the model you've pre-trained on https://huggingface.co/models? I saw you've recently set up the GitHub repository for MonoArt and I'm looking forward to the release!

Hosting on Hugging Face will give you more visibility and enable better discoverability. We can add tags like `image-to-3d` in the model cards so that people find the models easier, link it to the paper page, etc.

If you're down, leaving a guide [here](https://huggingface.co/docs/hub/models-uploading). If it's a custom PyTorch model, you can use the [PyTorchModelHubMixin](https://huggingface.co/docs/huggingface_hub/package_reference/mixins#huggingface_hub.PyTorchModelHubMixin)
class which adds `from_pretrained` and `push_to_hub` to the model which lets you to upload the model and people to download and use models right away.

After uploaded, we can also link the models to the paper page (read [here](https://huggingface.co/docs/hub/en/model-cards#linking-a-paper)) so people can discover your work.

You can also build a demo for your model on [Spaces](https://huggingface.co/spaces), we can provide you a ZeroGPU [grant](https://huggingface.co/docs/hub/en/spaces-gpus#community-gpu-grants), which gives you A100 GPUs for free.

Let me know if you're interested/need any guidance :)

Kind regards,

Niels

**Comment by animemory** [comment]
> Hi Niels,
> 
> Thanks for your interest in our work! We will release the pretrained model and code on HuggingFace ASAP.
> 
> Best, Haozhe.

Hi, any time scheduler?

**Comment by XiruiTeng** [comment]
> > Hi, any time scheduler?
> 
> Thanks for your interest! We plan to release the code and pretrained models after the paper is accepted. Please stay tuned.

Hi, any time scheduler?

**Comment by Quest4Science** [comment]
您好，来信已收到，我会尽快回复。祝您天天开心 (*^▽^*)

**Comment by Quest4Science** [comment]
Hi everyone,

Thanks for your patience! We have just released an initial version of the code and checkpoints. Please check the README for details.

We will continue to refine and update the repository. Closing this issue for now.

Best,
Haitian

**Comment by NielsRogge** [comment]
Hey @Quest4Science, congratulations on the initial release and on the ECCV 2026 acceptance.

Since the checkpoints are currently hosted as GitHub release assets, would you also consider uploading them to the Hugging Face hub (https://huggingface.co/models)? Hosting there gives you more visibility and discoverability, and once the model repo mentions the paper (preferably https://huggingface.co/papers/2603.19231, otherwise the arXiv abstract or PDF URL in the model card README), it will automatically show up on the paper page so people can find the checkpoints from there.

If it helps, you can use PyTorchModelHubMixin to add from_pretrained and push_to_hub to a custom PyTorch model: https://huggingface.co/docs/huggingface_hub/package_reference/mixins#huggingface_hub.PyTorchModelHubMixin

Thanks!
